configuration/pipeline_config.json

```diff
@@ -40,6 +40,13 @@
         "3903ad8e-284e-48a0-881b-89c0b06b9334",
         "45939459-e4d5-4ad6-8650-3e7134b9a035"
       ]
+    },
+    {
+      "SourceType": "RecoveryCSVURLs",
```

Comment:
Is this right? I thought this was called RecoveryCSV now

Reply:
Added this from memory - Would you be having an example of a recent project you did this - that has new configurations?